### DISCUSSION TITLE
Feature Request: Provide `default_baudrate` setting in user config file `.avrduderc`

### DISCUSSION THREAD

**Discussion by Ho-Ro** [avrdudes/avrdude#1589] [Ideas]
You can set `default_programmer` and `default_serial` in your personal config file `~/.avrduderc`, but `baudrate` must be set explicitly with the command line option `-b <baudrate>`.

**Comment by mcuee** [comment]
I think this is a good idea. PR will be welcome.

**Comment by Ho-Ro** [comment]
Here we are: #1590
